### ISSUE TITLE
how to use Cython to build fastapi and pydantic project

### ISSUE THREAD

**Issue by panla** [tiangolo/fastapi#3871]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
@app.get('/api/index/1')
def get(token: = str Header(..., description='token')):
    return {"success": True}
```

### Description

after use Cython to transform it to .so

run it will raise

- TypeError: Expected unicode, got Header

### Operating System

Linux

### Operating System Details

deepin

### FastAPI Version

0.68

### Python Version

3.8.11

### Additional Context

- Cython: 3.0.0a9
- pydantic: 1.8.2

**Comment by labeneator** [comment]
Hi @panla,
 How were you able to resolve this issue?

**Comment by panla** [comment]
> Hi @panla, How were you able to resolve this issue?

No, Use Cython, we can translation simple code, but can`t deal complex code, 
and, we can not rewrite exists code to accommodate Cython.
